eng/versioning/external_dependencies.txt

```diff
@@ -183,8 +183,8 @@ com.microsoft.azure:azure-storage;8.0.0
 com.microsoft.azure:msal4j;1.11.0
 com.microsoft.azure:msal4j-persistence-extension;1.1.0
 com.sun.activation:jakarta.activation;1.2.2
-io.opentelemetry:opentelemetry-api;1.0.0
-io.opentelemetry:opentelemetry-sdk;1.0.0
+io.opentelemetry:opentelemetry-api;1.10.0
+io.opentelemetry:opentelemetry-sdk;1.10.0
 io.opentelemetry:opentelemetry-exporter-logging;1.0.0
 io.opentelemetry:opentelemetry-exporter-jaeger;1.0.0
```

Comment:
@trask should I also update these 2 packages?

Reply:
yes 👍